src/autonlp/evaluate.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Tuple
+
+from .tasks import TASKS
+from .utils import BOLD_TAG, GREEN_TAG, RESET_TAG, YELLOW_TAG
+
+
+JOB_STATUS = (
+    ("⌚", "queued"),
+    ("🚀", "start"),
+    ("⚙", "data_munging"),
+    ("🏃", "model_evaluating"),
+    ("✅", "success"),
+    ("❌", "failed"),
+)
+
+
+def get_task(task_id: int) -> str:
+    for key, value in TASKS.items():
+        if value == task_id:
+            return key
+    return "❌ Unsupported task! Please update autonlp"
+
+
+def get_eval_job_status(status_id: int) -> Tuple[str, str]:
+    try:
+        return JOB_STATUS[status_id - 1]
+    except IndexError:
+        return "❓", "Unhandled status! Please update autonlp"
```

Comment:
This is from `project`, let's move them to a `utils` package and reuse them.

Please avoid copy-pasting things that don't need to! It's better to have a single piece of code to manage